# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: third_party/rust/cubeb-coreaudio/.travis.yml
# Commit: 24b97b0c24f0
# Full Hash: 24b97b0c24f0f30965d5d30dfa2a796e3c994e5e
# Author: Chun-Min Chang <chun.m.chang@gmail.com>
# Date: 2020-02-13 21:42:57
# Description:
#   Bug 1613156 - Update cubeb-coreaudio to 7fe03b4. r=padenot
#   
#   Pick commits:
#   - 7fe03b4: Bail out if the output device has no output channel (#50)
#   - ad74bad: Update README
# ==============================================================================

diff -r 8e18b9d4caae -r 24b97b0c24f0 third_party/rust/cubeb-coreaudio/.travis.yml
--- a/third_party/rust/cubeb-coreaudio/.travis.yml	Thu Feb 13 16:09:09 2020 +0000
+++ b/third_party/rust/cubeb-coreaudio/.travis.yml	Thu Feb 13 15:52:30 2020 +0000
@@ -11,3 +11,8 @@
 script:
   - cargo build --verbose
   - sh run_tests.sh
+  - sh run_sanitizers.sh
+jobs:
+  allow_failures:
+    - rust: nightly
+  fast_finish: true